restli-client/src/main/java/com/linkedin/restli/client/util/FluentClientUtils.java

```diff
@@ -61,4 +66,22 @@ private FluentClientUtils()
     }
     return input;
   }
+
+  public static Map<FieldDef<?>, Object> buildReadOnlyActionParameters(Map<FieldDef<?>, Object> actionParams)
+  {
+    return ActionRequestBuilder.buildReadOnlyActionParameters(actionParams);
```

Comment:
We haven't been building readonly parameters in the fluent client. Since we don't create an intermediate request builder, there is really no need to build them.